Do not return `<none>` placeholders for images any more (#4673)
cri-o/cri-o: README.md

Please refer to the [CRI-O Metrics guide](tutorials/metrics.md).

#### Container Runtime Interface special cases

Some aspects of the Container Runtime are worth some additional explanation.

> We summarized those information in a [dedicated guide](cri.md).

Reviewer: ```suggestion
These details are summarized in a [dedicated guide](cri.md).
```
